docs/user-guide/install-zos.md

```diff
@@ -176,7 +176,7 @@ To install Zowe API Mediation Layer, Zowe Application Framework, and z/OS Servic
 
     ```yaml
     install:
-     rootDir=/var/zowe/1.3.0
+     rootDir=/tmp/zowe/1.3.0
```

Comment:
why wouldn't you suggest /usr/lpp/zowe/1.3.0?

Reply:
For SMP/E we are going to use /usr/lpp/zowe/v1 and update this between versions, so we don't keep 1.2 and 1.3 and 1.4 separate, rather we always replace /usr/lpp/zpwe/v1.  I didn't want to suggest this before we have SMP/E working as we don't want customers using the simple zowe-install.sh to push this to /usr/lpp/zowe/v1 as 1) they probably won't have permission and 2) we want that reserved for SMP/E so that the CSI installs into that location and doesn't want anyone else to use that.